libraries/AP_Common/Location.cpp

```diff
@@ -5,7 +5,6 @@
 #include "Location.h"
 
 #include <AP_AHRS/AP_AHRS.h>
-#include <AP_AHRS/AP_AHRS_NavEKF.h>
```

Comment:
How is it useless? There is a pointer to it AP_AHRS_NavEKF... If anything, AP_AHRS.h seems pointless.

Reply:
because it is already include by ap_ahrs.h